source/_integrations/transmission.markdown

```diff
@@ -20,6 +20,11 @@ ha_integration_type: integration
 
 The Transmission integration allows you to monitor your [Transmission](https://www.transmissionbt.com/) BitTorrent downloads from within Home Assistant and set up automations based on that information.
 
+<div class='note warning'>
+This integration is compatible with Transmission client versions 1.31 - 2.82.
+Transmission 3.0 is NOT supported!
```

Comment:
Please list what is supported, as normal text, instead of writing what isn't supported.
The use of capital words `NOT` and exclamation marks isn't needed.